printer_mods/README.md

```diff
@@ -112,6 +112,8 @@ like so:
 | evandepol | [Crydom GN Terminal Guard](./evandepol/Crydom-GN-Guard) | A simple safety guard for the terminals on a Crydom GN SSR | :x: | :x: | :heavy_check_mark: |:x:|
 |  | [Voron 2.4 cable clamps](./evandepol/Voron2.4-Cable-clamps) | A set of 2020-mounted cable guards to organize and dress the wiring on your build | :x: | :x: | :heavy_check_mark: |:x:|
 | falo | [Magnetic Exhaust Grill Cover](./falo/magnetic_grill_cover) | A magnetic cover to seal the filtered exhaust | :x: | :x: | :heavy_check_mark: | :x: |
+| Fiction | [Exhaust Cover SFS](./Fiction/Exhaust_cover_SFS) | Exhaust cover and BTT Smart Filament Sensor Mount | :x: | :x: | :heavy_check_mark: | :x: |
+| | [C270 mount](./Fiction/C270_mount) | Logitech C270 webcam mount | :x: | :x: | :heavy_check_mark: | :x: |
```

Comment:
I count 4 mods in this PR and this only lists 2.

Reply:
I have added rows for the five mods

> Thanks for your mod!
> 
> There are some things I see that would need to be addressed. I noticed there are multiple mods included in this PR. While not a dealbreaker, we will be reviewing each individually but all have to be good for the PR to be accepted.
> 
> 1. There are two files named "Screenshot 2021-11-08" that are in the root directory.  Those need to be removed or moved under your directory.
> 2. The C270 readme and the LED profile mount readme needs more context than just a picture or two.  It need some explanation of what it's for and how it would be used.
> 3. In printer_mods/Fiction/Crossrail/STLs/crossrail.stl and printer_mods/Fiction/Crossrail/STLs/crossrail_slot.stl there are unsupported counterbores that will make a mess if printed like that.  These need to be stepped holes in the style seen in Trident or have a large chamfer.
> 4. While I may not consider it a major issue, I would strongly recommend giving some of the image files named like "printer_mods/Fiction/Crossrail/images/_DSF0724.jpg" a more descriptive name.
> 5. The file printer_mods/Fiction/Exhaust_cover/STLs/.foo.txt needs to be removed.
> 6. The exhaust cover readme (both of them) needs a sentence or two of explanation at the top of the purpose of the mod.
> 7. The updates to another user's repos must be removed.  (printer_mods/deepfriedheroin/v2_bed_wagos/images/Bed_wagos_assembly.png and others)

I apologise for the multiple mods in one pull request and incomplete readme files. I have to admit I didn't realise that changes made after the PR would be included.

1. Deleted the extraneous files
2. Added more information to the readme
3. Added chamfers to counterbores on the underside
4. Renamed photos to be more descriptive and updated readme(s) to match
5. Deleted
6. Added more information to the readme(s) here too
7. Deleted these files as well